### PR TITLE
Removes sec access + miranda from inspector

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
This pr removes the miranda ability and security access from inspectors. Also changes the description of the inspectors coat to fit them better.

## Why's this needed? 
Inspector isnt security, having them have these things I think sends the wrong message, nor do they help much with the gimmicl of the job.

## Changelog

```changelog
(u)Ikea
(+)Inspector no longer has security access
```

### CODE REVIEW

**Comment by blackrep** [comment]
The PR title accurate "Removes sec access"? Reading over the changes, it doesn't touch access at all.

**Comment by Momoberry** [comment]
Honestly inspector should have security access, but I don't believe they should have miranda rights.

File: code/obj/item/clothing/armor.dm [hunk 1]
```diff
@@ -406,7 +406,7 @@
 
 /obj/item/clothing/suit/armor/NT
 	name = "armored nanotrasen jacket"
-	desc = "An armored jacket worn by NanoTrasen security commanders."
+	desc = "An armored jacket given by nanotrasen for when their bureaucrats inevitably get shoved into hostile situations."
```
Comment (TobleroneSwordfish) [inline code/obj/item/clothing/armor.dm:409]: This is used for other roles too, would be better to make a variant for inspector

Comment (ThatFiveGuys) [reply code/obj/item/clothing/armor.dm:409]: The only other thing its used for is a decrepit old nt role that i have never seen used

**Comment by ThatFiveGuys** [comment]
> The PR title accurate "Removes sec access"? Reading over the changes, it doesn't touch access at all.

Yeah forgot to commit it, working on another thing and will commit it once i commit that

**Comment by ThatFiveGuys** [comment]
> Honestly inspector should have security access, but I don't believe they should have miranda rights.

I think the sec access + the badge can lead to people treating it like a sec role, and Id much rather have the sec access removed then the badge removed because the badge is cool.

**Comment by Momoberry** [comment]
> > Honestly inspector should have security access, but I don't believe they should have miranda rights.
> 
> I think the sec access + the badge can lead to people treating it like a sec role, and Id much rather have the sec access removed then the badge removed because the badge is cool.

The badge seems kind of pointless if it's representing nothing. Maybe resprite the badge to be a unique version then, cause it's just the same badge security uses, and it's confusing if they can also get the badge but can't go into security.

**Comment by ThatFiveGuys** [comment]
Im mostly just against no timelock gimmick roles getting sec access in general, the badge is a cute thing to represent your offical nanotrasen status. Removing sec access wont be a hill ill die on however just think itd be a good thing as it doesnt remove much from the game and it helps prevent a lot of annoying stuff to deal with as sec

**Comment by Exester509** [comment]
I'd say that Inspectors should only have as much sec access as detectives do. Which is basically flashes, cuffs, and you could get a helmet from a vending machine.

**Comment by ThatFiveGuys** [comment]
> I'd say that Inspectors should only have as much sec access as detectives do. Which is basically flashes, cuffs, and you could get a helmet from a vending machine.

Why do inspectors need any of these things theyre bureacrats there to inspect, not cops

**Pull request closed without merging**